eclipse-collections-code-generator/src/main/resources/test/list/immutable/immutablePrimitiveSingletonListTest.stg

```diff
@@ -102,6 +104,13 @@ public class Immutable<name>SingletonListTest extends AbstractImmutable<name>Lis
         Assert.assertEquals(0, this.classUnderTest().binarySearch(<(literal.(type))("1")>));
         Assert.assertEquals(-2, this.classUnderTest().binarySearch(<(literal.(type))("5")>));
     }
+
+    @Test
+    public void toStack()
+    {
+        Mutable<name>Stack stack = this.classUnderTest().toStack();
+        Assert.assertEquals(stack, <name>Stacks.mutable.with(<(literal.(type))("1")>));
```

Comment:
The arguments in the `assertEquals` should be reversed (expected, equals)

Reply:
I have reversed the values for assertEquals.